src/ukify/ukify.py

```diff
@@ -1584,6 +1588,12 @@ def __call__(
         config_key='UKI/Initrd',
         config_push=ConfigItem.config_list_prepend,
     ),
+    ConfigItem(
+        '--firmware',
+        type=Path,
+        help='firmware file [part of .fmw section]',
```

Comment:
"part of" seems to be copy pasta. `--firmware` can appear only once.

Reply:
Fixed.